certificate/models.py

```diff
@@ -269,3 +270,18 @@ class Internship16_participant(models.Model):
     project_title = models.CharField(max_length=1000)
     internship_project_duration = models.CharField(max_length=500, null=True, blank=True)
     purpose = models.CharField(max_length=10, default='F16')
+
+
+attendee_types = (
+    ('P','Participants'),
+    ('A','Speaker'),
+    ('W','Workshop'),
+    ('T','Organizers')
+    )
+
+class Scipy_2017(models.Model):
+    name = models.CharField(max_length=300)
+    email = models.CharField(max_length=300)
+    paper = models.CharField(max_length=300)
+    purpose = models.CharField(max_length=10, default='S17')
+    attendee_type = models.CharField(max_length=25, choices=(attendee_types)
```

Comment:
Closing bracket missing.
attendee_types should not be in parentheses

Reply:
done